### ISSUE TITLE
About the training problem of yolov5s and yolov5m

### ISSUE THREAD

**Issue by Zengyf-CVer** [ultralytics/yolov5#6980]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

@glenn-jocher 
I did an experiment like this on a custom dataset. First, I used yolov5s as the pre-training weight, the batch size was 64, and the epochs was 200. In the second experiment, I used yolov5m as the pre-training weight, and the batch size was 32. , with epochs of 200. Interestingly, the experiment trained with yolov5m has reached saturation around the 90th epoch, and the corresponding mAP value is not as high as that of the experiment with yolov5s. Why? I had the same problem with another custom dataset before. According to common sense, the pre-trained weights and architecture of yolov5m should have higher mAP values than the pre-trained weights and architecture of yolov5s, but the opposite is true, and I don't know what's going on.

This is the training process of `yolov5s`, the AP at 91epoch is 0.4227, and the AP50 is 0.7565:
![ksnip_20220315-104603](https://user-images.githubusercontent.com/41098760/158296882-132686ba-0755-428b-89dc-52c2b0bccb2f.png)

This is the training process of `yolov5m`, the AP at 91epoch is 0.4088, and the AP50 is 0.7309:

![ksnip_20220315-104544](https://user-images.githubusercontent.com/41098760/158296952-fff4e302-6d9a-4786-9b62-878d30c57417.png)

It seems that the mAP of yolov5m in the first few epochs is higher than that of yolov5s, but the mAP in the later epochs is lower than that of the latter.

**Comment by glenn-jocher** [comment]
@Zengyf-CVer that's very interesting. There could be a few possibilities:

- Smaller models tend to produce better results early on but have less overall learning capability and so produce worse results in later training. Perhaps you haven't trained long enough (are val losses overfitting?)
- Larger models benefit from additional augmentation, i.e. a more aggressive hyperparameter augmentation file.
- The problem may just be very simple or dataset very small and you are just seeing random noise in training results.

**Comment by Zengyf-CVer** [comment]
@glenn-jocher 
Thanks a lot, I will look into it further.

**Comment by Zengyf-CVer** [comment]
@glenn-jocher 
I re-checked the val loss of both and found that the loss of yolov5m has an upward trend after 70 epochs, while that of yolov5s has no such trend.

- yolov5m val obj loss:
![widerface-m_val_objLoss](https://user-images.githubusercontent.com/41098760/159145270-5faffa7a-77f1-4d12-a1c7-282c62287b2a.png)

- yolov5s val obj loss:
![widerface-s_val_objLoss](https://user-images.githubusercontent.com/41098760/159145273-4d3fadb7-6edf-402f-84d6-c1055e3042e7.png)

**Comment by Zengyf-CVer** [comment]
@glenn-jocher 
Which augmentation method is more appropriate for the pretrained weight `yolov5m`?
![ksnip_20220320-103045](https://user-images.githubusercontent.com/41098760/159145450-f7b0256c-b7e6-4631-9bf7-6561ef3d2c7e.png)

**Comment by glenn-jocher** [comment]
@Zengyf-CVer -med or -low work well with YOLOv5m and smaller models. They're pretty similar though so I wouldn't expect switching -low for -med hyp yamls to have a huge effect.

**Comment by Zengyf-CVer** [comment]
@glenn-jocher 
Thank you very much.
